src/ontology/mondo-edit.obo

```diff
@@ -317208,10 +317184,9 @@ xref: NCIT:C98622 {source="MONDO:equivalentTo"}
 xref: Orphanet:2004 {source="MONDO:equivalentTo"}
 xref: SCTID:232461002 {source="MONDO:equivalentTo"}
 xref: UMLS:C1840311 {source="MONDO:notFoundInDiseaseSubset", source="NCIT:C98622"}
-is_a: MONDO:0015207 {source="Orphanet:2004"} ! non-syndromic esophageal malformation
-is_a: MONDO:0015221 {source="Orphanet:2004"} ! non-syndromic respiratory or mediastinal malformation
-is_a: MONDO:0015504 {source="Orphanet:2004"} ! larynx anomaly
-is_a: MONDO:0015930 {source="Orphanet:2004"} ! respiratory malformation
+is_a: MONDO:0003749 {source="https://orcid.org/0000-0001-5208-3432"} ! esophageal disorder
+is_a: MONDO:0015504 {source="Orphanet:2004", source="https://orcid.org/0000-0001-5208-3432"} ! larynx anomaly
+relationship: has_characteristic MONDO:0021136 ! rare
```

Comment:
did you add this manually?

Reply:
yes, the green I either added (line 317187) or added my ORCID (in line 317188) and line 317189 came from rewiring and I left it. I figured those will be addressed (and sources added) later?

the red were added due to rewiring and i removed them (with the exception of line 317213, larynx anomaly, which I added my ORCID to, because I agreed with it.